.env.demo

```diff
@@ -0,0 +1,15 @@
+VUE_APP_ENVIRONMENT=demo
```

Comment:
I don't think we'll accomodate for a separate `demo` environment, just use staging here.